dev-util/gammaray/gammaray-2.9.0.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+QT_MINIMAL="5.9"
+
+DESCRIPTION="High-level runtime introspection tool for Qt applications"
+HOMEPAGE="https://github.com/KDAB/GammaRay \
+		https://www.kdab.com/?s=gammaray"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
+SLOT=0
+IUSE="3d auto-detect backtrace bluetooth +client \
+	concurrent designer -doc feedback geolocation +gpl-only-features \
+	printsupport scxml svg test qml unwind \
+	wayland webengine"
+
+RDEPEND="
+	$(add_qt_dep qtcore)
+	$(add_qt_dep qtgui)
+	$(add_qt_dep qtnetwork)
+	$(add_qt_dep qtwidgets)
+	$(add_qt_dep qtxml)
+	$(add_frameworks_dep kitemmodels)
+"
+
+DEPEND="${RDEPEND}
+	3d? ( $(add_qt_dep qt3d) )
+	bluetooth? ( $(add_qt_dep qtbluetooth) )
+	concurrent? ( $(add_qt_dep qtconcurrent) )
+	designer? ( $(add_qt_dep designer) )
+	geolocation? ( $(add_qt_dep qtpositioning) )
+	printsupport? ( $(add_qt_dep qtprintsupport) )
+	qml? ( $(add_qt_dep qtdeclarative) )
+	scxml? ( $(add_qt_dep qtscxml) )
+	svg? ( $(add_qt_dep qtsvg) )
+	test? ( $(add_qt_dep qttest) )
+	webengine? ( $(add_qt_dep qtwebengine 'widgets') )
+	wayland? ( $(add_qt_dep qtwayland) )
+"
+
+src_configure(){
+	#Upstream uses a *ton* of automagic deps
+	local mycmakeargs+=(
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt53DAnimation=$(usex !3d)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt53DExtras=$(usex !3d)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt53DInput=$(usex !3d)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt53DLogic=$(usex !3d)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt53DRender=$(usex !3d)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt53DQuick=$(usex !3d)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt5Bluetooth=$(usex !bluetooth)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt5Concurrent=$(usex !concurrent)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt5Designer=$(usex !designer)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt5Positioning=$(usex !geolocation)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt5PrintSupport=$(usex !printsupport)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt5Qml=$(usex !qml)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt5Quick=$(usex !qml)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt5QuickWidgets=$(usex !qml)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt5Script=ON
```

Comment:
Why do you disable it (+line below)? (these would be pulling in `qtscript`)

Reply:
I thought it means means the other way around. That it wouldn't search for it, hence it wouldn't install it as well.

Reply:
yes, that's my question, why you do it. (these would be pulling in `qtscript` [if not disabled here])

Reply:
Because `qtscript` is supposed to be deprecated. It doesn't bother me to enable it.

Reply:
Fine by me then. (I see you removed the lines since then, but that introduces an automagic dep which we can't have). A lot of ebuilds still require qtscript, but you will understand better if it is important enough to add another use flag for it.